lib/unwrappr/cli.rb

```diff
@@ -17,6 +17,13 @@ class CLI < Clamp::Command
            DESCRIPTION
            attribute_name: :base_branch)
 
+    option ['-f', '--lock-file'],
+           'LOCK_FILE',
```

Comment:
Let's clarify how to specify multiple filenames:

```suggestion
           'LOCK_FILE1[,LOCK_FILE2][,LOCK_FILE3]',
```

Reply:
Or is it 
```suggestion
           LOCK_FILE1 [-f LOCK_FILE2] [-f LOCK_FILE3] [-f ...]',
```

Reply:
It's the latter.